Source/Core/DolphinQt/Config/LogConfigWidget.cpp

```diff
@@ -136,6 +142,7 @@ void LogConfigWidget::LoadSettings()
   m_verbosity_error->setChecked(verbosity == 2);
   m_verbosity_warning->setChecked(verbosity == 3);
   m_verbosity_info->setChecked(verbosity == 4);
+  m_verbosity_debug->setChecked(verbosity == 5);
```

Comment:
These could be replaced with `verbosity == LogTypes::LDEBUG`, `verbosity == LogTypes::LINFO`, etc. I know it's in the existing code, but may as well clean it up while you're changing it.